### PR TITLE
Component Library RathUI

### PR DESCRIPTION
**Pull request by virendrawadher**
Hello guys, this is the intial version of rathUI which is the component library. It will be pleasure if you take your sometime and review the code, it will help me to improve my code.
RathUI:- https://rathui.netlify.app

### CODE REVIEW

**Comment by soham-parab** [comment]
Awesome and clean UI! Loved the font selection and especially liked the alerts and cards.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            </div>
+            <div class="mobile-nav__heading" href="../index.html">
+                RathUI
+            </div>
+        </div>
+    </header>
+    <div class="main-content">
+        <div class="homepage">
+            <span class="tagline">Component Library for faster and easier web development.</span>
+            <button class="gs-button"><a href="/installation/installation.html">Get Started</a> <i class="fa fa-arrow-right fa-2x"></i> </button>
```
Comment (soham-parab) [inline index.html:40]: Navigation to the next page after 'get started' only happens when I click on the text 'get started' and not the button area around it.

File: image/image.html [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+                    <ul class="mobile-menu__subitem">
+                        <li><a href="/alert/alert.html">Alert</a></li>
+                        <li><a href="/avatar/avatar.html">Avatar</a></li>
+                        <li><a href="/badge/badge.html">Badge</a></li>
+                        <li><a href="/Button/button.html">Button</a></li>
+                        <li><a href="/card/card.html">Card</a></li>
+                        <li><a href="/form/form.html">Form</a></li>
+                        <li><a href="/image/image.html">Image</a></li>
+                        <li><a href="/input/input.html">Input</a></li>
+                        <li><a href="/list/list.html">List</a></li>
```
Comment (soham-parab) [inline image/image.html:57]: The color contrast between the text color and background color is poor, therefore difficult to read. Maybe choose a darker grey.

File: avatar/avatar.html [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+                </div>
+                <div class="avatar">
+                    <span class="name">Avatar Sizes</span>
+                    <div class="avatar-group">
+                        <img src="https://i0.wp.com/therighthairstyles.com/wp-content/uploads/2014/03/24-side-bangs.jpg?w=500&ssl=1" alt="Manika" class="extra-small">
+                        <span class="small">
+                            <div class="tp-small">TP</div>
+                        </span>
+                        <img src="https://avatars0.githubusercontent.com/u/1500684?s=460&v=4" alt="Git Person" class="medium">
+                        <img src="https://avatars3.githubusercontent.com/u/100200?s=460&v=4" alt="Paul" class="large">
```
Comment (soham-parab) [inline avatar/avatar.html:109]: Adding some padding here would help the images breathe.

File: card/card.html [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+                                    <span class="vertical pt3"></span>
+                                </div>
+                            </div>
+                            <img src="https://images.unsplash.com/photo-1574126154517-d1e0d89ef734?ixlib=rb-1.2.1&ixid=MXwxMjA3fDB8MHxleHBsb3JlLWZlZWR8NHx8fGVufDB8fHw%3D&auto=format&fit=crop&w=1000&q=60);" alt="" class="card-image">
+                            <div class="card-content">
+                                <p>This impressive paella is a perfect party dish and a fun meal to cook together with your guests. Add 1 cup of frozen peas along with the mussels, if you like.</p>
+                            </div>
+                            <div class="footer">
+                                <i class="fa fa-heart fa-2x"></i>
+                                <i class="fa fa-share-alt fa-2x"></i>
```
Comment (soham-parab) [inline card/card.html:146]: Adding "cursor: pointer" for these buttons would be a good idea.

File: form/form.html [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+                    <label class="label-apple">Apples</label>
+                    <input type="radio" name="furits">
+                    <label>Oranges</label>
+                    <input type="radio" name="furits">
+                    <label>Pears</label>
+                    <input type="radio" name="furits">
+                    <label>Grapes</label>
+                    <input type="radio" name="furits">
+                    <label>Mango</label>
+                </div>
```
Comment (soham-parab) [inline form/form.html:150]: Please recheck if the radio buttons are working.

File: form/form.html [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+                    </iframe>
+                </div>
+            </div>
+            <div class="forms">
+                <span class="name">Text Area</span>
+                <div class="box-border">
+                    <label class="text">Text</label>
+                    <textarea class="text-textarea"></textarea>
+                    <label class="short-text">Short Text</label>
+                    <textarea class="short-text-textarea"></textarea>
```
Comment (soham-parab) [inline form/form.html:182]: Please recheck if the textarea is working when clicked on.

**Review by soham-parab — COMMENTED** [review]
Great job and a ton of effort! Please consider making the suggested changes, thank you!

File: avatar/avatar.html [hunk 7]
```diff
@@ -0,0 +1,182 @@
...
+                            style="width: 100%; height: 370px; border:0; transform: scale(1); overflow:hidden;"
+                            sandbox="allow-scripts allow-same-origin">
+                        </iframe>
+                    </div>
+                </div>
+                <div class="avatar ag">
+                    <span class="name">Avatar Group</span>
+                    <div class="avatar-group">
+                        <div class="avatars-group">
```
Comment (Sumanbitw) [inline avatar/avatar.html:161]: Give some margin to avatar groups

**Review by Sumanbitw — COMMENTED** [review]
Great job !  Really loved the design .

**Pull request merged**